dcos/dcos-ui
# [DCOS-49636] Enable UI prefix label for SDK packages

## Testing

* create marathon app having ui labels -> link should be displayed
* install non-sdk package with ui -> link should be displayed
* install sdk package without label -> NO Link should be displayed
* install sdk package with label -> Link from label should be displayed



## Trade-offs

This PR consists of three commits, which I decided to put into one PR:
1. refactoring service util in general to remove duplicate from `src/js/utils` and resolve circular dependency in `plugins/src/js/utils`
2. refactoring structs and adding functionality to enable new label usage (this is one commit b/c I wasn really able to split it up without creating broken code)
3. small ux fix to always display webui icon



## Dependencies

None


## Screenshots
